watsononlinestore/watson_online_store.py

```diff
@@ -83,6 +85,46 @@ def handle_db_lookup(self):
             print("DB.\n context:{}\n".format(self.context))
         return False
 
+    def handle_lookupAndAddEmail(self):
+        """ Verify email is not in DB and add to DB.
+        """
+        email_addr = str(self.context['email'])
+        if "mailto" in email_addr:
+            email_addr = self.cleanup_email(email_addr)
+        if DEBUG:
+            print("DB.\n email_addr:{}\n".format(email_addr))
+        existing_user = self.cloudant_online_store.find_customer(email_addr)
+        if DEBUG:
+            print("DB.\n existing_user:{}\n".format(existing_user))
```

Comment:
Yeah, that crap is mostly in there for my own debugging. It'll mostly go, but I reckon we'll have some minimal logging. That's for the future....